### ISSUE TITLE
Understand Query-Index right - or Failed Query Index differs

### ISSUE THREAD

**Issue by simeonackermann** [AKSW/IGUANA#59]
It seems the query index from console log and results file differs.
I tested 100 queries on virtuoso and got messages like:

```
| [WARNING  Sun Jul 10 13:21:25 UTC 2016] Query 13 wasn't successfull for connection store_virtuoso_1. See logs for more inforamtion
| [WARNING  Sun Jul 10 13:21:25 UTC 2016] This will be saved as failed query
...
```

This happens for queries: 13, 19, 20, 23, 24, 25 what seems totally ok.
But in results file `Failed_Queries` is:

| Connection | 0 | 1 | 2 | ... | 13 | 14 | 15 | 16 | ... | 19 | 20 | 21 | 22 | 23 | 24 | 25 | ... | 31 | 32 | 33 | 34 | 35 | 36 | 37 | 38 | ... |
|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
| store_virtuoso_1 | 0 | 0 | 0 | ... | 0 | 0 | 146 | 0 | ... | 0 | 0 | 146 | 0 | 0 | 0 | 0 | ... | 0 | 146 | 0 | 0 | 146 | 146 | 146 | 0 | ... |

So the index of failed queries differs from my console output.

I wondering whats the right index and why these differs...?

**Comment by TortugaAttack** [comment]
Hey. Sorry for the late response, but I am currently in vacation.
Can you provide your config to me (either here or per Mail) and best case the queries too? So I can debug it. Because it should be the same Index. So if the Logs says 23 it should  be Query 23 ;)

**Comment by simeonackermann** [comment]
No problem! You should  stay in vacation without work!!

After that, here my queries (auto-generated with duplicates) and config (ip is from docker) 
[queries.txt](https://github.com/AKSW/IGUANA/files/363334/queries.txt)
[config.xml.txt](https://github.com/AKSW/IGUANA/files/363347/config.xml.txt)

**Comment by simeonackermann** [comment]
Another strange and I think new issue is the console output at the end of the test:
```
[SEVERE  Thu Jul 14 09:37:21 UTC 2016] Result size differs!!! 
[SEVERE  Thu Jul 14 09:37:21 UTC 2016] Old size : 18
[SEVERE  Thu Jul 14 09:37:21 UTC 2016] New size: 0
```

**Comment by TortugaAttack** [comment]
Okay. I will do that. 
Thanks for the data and queries. :)

The outpot is a debug Output which can be safely ignored ;)
I will remove the output in the next Release ;)

**Comment by TortugaAttack** [comment]
Okay. I tried it and it does actual work for me. 
So one more questions:
Which version of IGUANA do you use?
(Strict void checking is on, is that right?)

**Comment by TortugaAttack** [comment]
Okay,
I must update my response: 
It does work for small query sets. While the complete set of yours makes the same fault. 
Do you use one of the newest version? Then i might have the answer.
I will upload a new version today. 
I am really sorry about this. But it only changes the Failed and thus also the succeded metrics!!!

**Comment by TortugaAttack** [comment]
Soo. 
Found the damn bug! 
Will update the version asap! 
Last tests are running and i hope at 16:00 the new version will be uploaded.
I am really sorry about this! 
They ordered the results wrong.

**Comment by TortugaAttack** [comment]
Fixed in 1.2.0R1

**Comment by simeonackermann** [comment]
Thanks a lot! Seems to work now!
